src/pages/AddItem/styles.css

```diff
@@ -1,8 +1,46 @@
+body {
+  background-color: #f2b807;
+}
```

Comment:
Please, take care with lines 1-3! 👀 

Maybe in this specific it doesn't cause a big problem, because you change `body` tag directly, it's fine 👍 ! . However, I think it's not a good idea to apply this kind of changes from component styles. It's possible a better approach from your `App.css` or `index.css`.

Reply:
Done...